dojo/pipeline.py

```diff
@@ -93,7 +93,20 @@ def update_product_access(backend, uid, user=None, social=None, *args, **kwargs)
                         # Make product member an Owner of the product
                         product_member.role = 4
                         product_member.save()
-
+                # Import tags and/or URL if necessary
+                for project in projects:
```

Comment:
Here we have a loop over GitLab projects within a loop over GitLab project names. Took me a few seconds to understand why. I think you have to look for the project with the current name. It would be much better to refactor the method slightly and convert line 83 to be `for project in project`. Then we could get rid of that inner loop.